graylog2-web-interface/src/components/streamrules/StreamRulesEditor.jsx

```diff
@@ -17,14 +18,35 @@ import StoreProvider from 'injection/StoreProvider';
 const StreamsStore = StoreProvider.getStore('Streams');
 const StreamRulesStore = StoreProvider.getStore('StreamRules');
 
+const StreamAlertHeader = styled.h4`
+  font-weight: bold;
+  margin: 0 0 12px;
+`;
+
+const MatchIcon = styled(Icon)(({ empty, matches }) => {
```

Comment:
This will throw a console warning when `empty` or `matches` are `true`, since styled-components passes all props through: https://github.com/styled-components/styled-components/issues/1198#issuecomment-336628848

Reply:
Good catch!